base/steer/FairTask.h

```diff
@@ -21,8 +21,6 @@
 #ifndef FAIRTASK_H
 #define FAIRTASK_H
 
-#include "FairRootManager.h"   // for FairRootManager
```

Comment:
The removal of FairRootManager include from the FairTask header file
forces few additions of same include in the PandaRoot project.
Is it beneficial to remove it here?

Reply:
I think such an addition in the changelog is not needed.
Since the FairRootManager is used in these PandaRoot tasks, the corresponding include should be there anyways.